src/addressBook/index.test.js

```diff
@@ -433,33 +466,34 @@ describe("saveContact", () => {
   test("it saves the contact and the people index", async () => {
     const fetch = jest.fn();
     const addressBookIri = "https://user.example.com/contacts";
+    const mockAddressBook = solidClientFns.mockSolidDatasetFrom(addressBookIri);
     const webId = "https://user.example.com/card#me";
     const schema = { webId, fn: "Test Person" };
-    const contactDataset = setThing(
-      createSolidDataset(),
-      createThing({
-        url: "https://user.example.com/contacts/People/1234/index.ttl",
-      })
-    );
-    const peopleIndexDataset = setThing(
-      createSolidDataset(),
-      createThing({ name: "this" })
-    );
-    const peopleDataset = setThing(
-      createSolidDataset(),
-      createThing({ name: "this" })
+    const contactDataset = mockPersonDatasetAlice();
+
+    const peopleIndexIri = `${addressBookIri}/people.ttl`;
+    const peopleIndexDataset = solidClientFns.mockSolidDatasetFrom(
+      peopleIndexIri
     );
 
+    jest.spyOn(solidClientFns, "getSourceUrl").mockReturnValue(addressBookIri);
+
+    jest
+      .spyOn(addressBookFns, "createContact")
```

Comment:
`createContact` isn't an asynchronous function that connects to the server, so you shouldn't need to mock it.